action/alarmWebAction.js

```diff
@@ -103,6 +103,7 @@ function main(params) {
                     }
                     newTrigger.cron = params.cron;
                     newTrigger.timezone = params.timezone;
+                    newTrigger.strict = params.strict === 'true';
```

Comment:
Hmm.. If there is no `strict` parameter passed, it becomes false.

It has to check null and undefined type:
```
newTrigger.strict = params.strict === 'true' || params.strict === undefined || params.strict == null;
```

Reply:
First, thank you for finding my mistake. Then how about this?
```suggestion
                    newTrigger.strict = params.strict !== 'false';
```